adapters/oidc/js/src/keycloak.js

```diff
@@ -445,6 +445,11 @@ function Keycloak (config) {
             url += '&ui_locales=' + encodeURIComponent(options.locale);
         }
 
+        if (options && options.acr) {
```

Comment:
Can you add this option to the TypeScript definitions as well? Please also document the use of this option there and in the Keycloak JS documentation.

Reply:
Forgot to update typescript. I will update this PR for it. For the documentation, I was planning to do it later together with some other changes related to step-up authentication (See description of this PR for the details), however I can do it now as keycloak.js documentation is quite isolated. So will send PR for the documentation now.